classes/asserter/exception.php

```diff
@@ -19,14 +19,15 @@ public function __construct(atoum\asserter $asserter, $message)
             $line = null;
             $function = null;
 
-            foreach (array_filter(debug_backtrace(false), function ($backtrace) use ($file) {
-                return isset($backtrace['file']) === true && $backtrace['file'] === $file;
-            }) as $backtrace) {
-                if ($line === null && isset($backtrace['line']) === true) {
+            $backtrace = $this->getBacktrace($test);
+            if ($backtrace !== null) {
+                $file = $backtrace['file'] ?? $file;
+
+                if (isset($backtrace['line']) === true) {
                     $line = $backtrace['line'];
                 }
 
-                if ($function === null && isset($backtrace['object']) === true && isset($backtrace['function']) === true && $backtrace['object'] === $asserter && $backtrace['function'] !== '__call') {
+                if (isset($backtrace['object']) === true && isset($backtrace['function']) === true && $backtrace['object'] === $asserter && $backtrace['function'] !== '__call') {
```

Comment:
I noticed that this cannot work. Indeed `debug_backtrace(false)` will not provide `object` (I did not changed that).
If I change the call to `debug_backtrace()`, messages are changed to:
```diff
 > There are 2 failures:
=> tests\units\MyClass::testMethod1():
- In file /path/to/tests/MyClass.php on line 106, boolean() failed: boolean(false) is not true
+ In file /path/to/tests/MyClass.php on line 106, boolean::isTrue() failed: boolean(false) is not true
 -Expected
 +Actual
 @@ -1 +1 @@
 -bool(true)
 +bool(false)
=> tests\units\MyClass::testMethod2():
- In file /path/to/tests/MyClass.php on line 55, array() failed: array(3) has size 3, expected size 12
+ In file /path/to/tests/MyClass.php on line 55, array::hasSize() failed: array(3) has size 3, expected size 12
```

Should I fix this too ?